### DISCUSSION TITLE
Lemmy

### DISCUSSION THREAD

**Discussion by justTOBBI** [catppuccin/catppuccin#2108] [Port Requests]
### Will you make the port?

- [X] Yes, I'm willing to make the port

### How can the suggested port be themed?

Lemmy can be themed [instance wide](https://join-lemmy.org/docs/administration/theming.html) or via stylus.

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by PoorPocketsMcNewHold** [comment]
One already made it. And can be applied instance wide, or via Userscripts if you compile it locally *(Might need to automate it then)* https://github.com/n3oney/catppuccin-lemmy
https://lemmy.ml/post/1422309 Also applied to OP instance https://lemmy.neoney.dev/
Hi @n3oney

**Reply by n3oney** [reply]
Hi, yes, the theme needs to be updated a bit for 0.18.0 though.
